### DISCUSSION TITLE
Cannot discover the Comfoconnect LAN C Bridge

### DISCUSSION THREAD

**Discussion by crouser7** [michaelarnauts/aiocomfoconnect#39] [Q&A]
Hello,

i am trying to discover my bridge, but i do not get any response.
I have a Zehnder ComfoAir Q350. The App shows:
Wifi: ok
Internet: ok
Cloud: not in use

interface in local network: ok
remote: ok
status: connected

I tried running the python script for discovery. I also tried sending the broadcast udp from a separate microcontroller, but no answers.

please help, i would like to integrate it into my smarthome, for more efficient airflow control, based on co2 and temp sensors.

**Comment by michaelarnauts** [comment]
You don't need the discovery if you just provide the IP of the bridge.

Are you sure that you are in the same subnet as the bridge?

**Reply by crouser7** [reply]
All right, yes i see that now. 
I will try to implement the further functionality now in my esp32. 
By the way, running the comman python -m aiocomfoconnect register --host 192.168. ... got me the error message wrong pin, where can i supply the correct pin?
